go.mod

```diff
@@ -29,10 +30,16 @@ require (
 require (
 	cloud.google.com/go v0.102.1 // indirect
 	cloud.google.com/go/iam v0.3.0 // indirect
+	github.com/Microsoft/go-winio v0.4.16 // indirect
```

Comment:
I never quite know what to do with all these go.mod changes. I am leaving this open as a reminder for me to follow up.